deeppavlov/models/ranking/matching_models/deep_attention_matching_network.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
```

Comment:
Add licence please

Reply:
I was not 100% sure, so I requested the information from authors. It will be clear very soon, in a few days. It seems that we should to wait some time until the approval.